File: Data/Sample-DotNet/InventoryController.cs
using Microsoft.AspNetCore.Mvc;
using System.Data.SqlClient;

namespace InventoryAPI.Controllers
{
    [ApiController]
    [Route("api/[controller]")]
    public class InventoryController : ControllerBase
    {
        [HttpGet("items")]
        public IActionResult GetItems()
        {
            string connectionString = "Server=localhost;Database=InventoryDB;User Id=sa;Password=your_password;";
            using (SqlConnection connection = new SqlConnection(connectionString))
            {
                connection.Open();
                // fetch items
            }
            return Ok(new[] { "item1", "item2" });
        }

        [HttpPost("add")]
        public IActionResult AddItem(string itemName)
        {
            // logic to add item
            return Ok("Item added: " + itemName);
        }
    }
}

File: Data/Sample-Java/InventoryController.java
@RestController
@RequestMapping("/api/inventory")
public class InventoryController {

    @Autowired
    private PricingService pricingService;

    @GetMapping("/items")
    public List<String> getItems() {
        // Simulate DB connection
        String connectionString = "jdbc:mysql://localhost:3306/inventorydb";
        return List.of("item1", "item2");
    }

    @PostMapping("/add")
    public String addItem(@RequestBody String itemName) {
        return "Added: " + itemName;
    }

    @GetMapping("/price")
    public double getPrice(@RequestParam String item) {
        return pricingService.fetchPrice(item);
    }
}
